src/main/java/org/opensearch/plugin/insights/core/service/categorizer/SearchQueryAggregationCategorizer.java

```diff
@@ -49,7 +49,7 @@ public void incrementSearchQueryAggregationCounters(
     private void incrementCountersRecursively(AggregationBuilder aggregationBuilder, Map<MetricType, Number> measurements) {
         // Increment counters for the current aggregation
         String aggregationType = aggregationBuilder.getType();
-        searchQueryCounters.incrementAggCounter(1, Tags.create().addTag(TYPE_TAG, aggregationType), measurements);
+        searchQueryCounters.incrementAggCounter(1, Tags.create().addTag(AGGREGATION_TYPE_TAG, aggregationType), measurements);
```

Comment:
The aggregation type will still say just range, right? Or we can tag in the dashboard that the "range" is for aggregation instead of query?

Reply:
Yup but the attribute/tag will be `type=range` for query and `agg_type=range` for aggregation